client.lua

```diff
@@ -421,7 +421,7 @@ RegisterNetEvent('ox_inventory:setPlayerInventory', function(drops, inventory, w
 						local id = GetPlayerFromServerId(currentInventory.id)
 						local ped = GetPlayerPed(id)
 						local pedCoords = GetEntityCoords(ped)
-						if not id or #(playerCoords - pedCoords) > 1.8 or not (ESX.PlayerData.job.name == 'police' or CanOpenTarget(ped)) then
+						if not id or (#(playerCoords - pedCoords) > 1.8 or not (ESX.PlayerData.job.name == 'police' or CanOpenTarget(ped)) and ESX.PlayerData.group ~= 'admin') then
```

Comment:
This would still do the distance check for admins? we kinda want admins to be able to access it globally